.gitmodules

```diff
@@ -2,13 +2,13 @@
 	path = 3rdparty/mshadow
 	url = https://github.com/dmlc/mshadow.git
 [submodule "dmlc-core"]
```

Comment:
why are some submodules like "3rdparty/openmp" and others are "ps-lite"? Is this just an identifier? Even so, should we standardize this?

Reply:
Good question. The name `ps-lite` remains unchanged when I use the `git mv` command to move the submodule to another folder. I agree that it makes sense to standardize. Let me try to update the name locally first